tests/models/unit_cases_gas/multi_demand_source_node/src/run_test.py

```diff
@@ -52,12 +62,22 @@ class GasProblem(
     ESDLMixin,
     CollocatedIntegratedOptimizationProblem,
 ):
-    def path_goals(self):
-        goals = super().path_goals().copy()
-
-        return goals
+    """
+    This is a problem to check the behaviour of a gas network with a node.
+    """
 
     def times(self, variable=None) -> np.ndarray:
+        """
+        We shorten the horizon here to have a faster test.
```

Comment:
Suggest 3rd person usage: "Shorten the time horizon ..."

Reply:
done